users/nazu/oled/oled_logo.c

```diff
@@ -0,0 +1,114 @@
+/*    static const char PROGMEM heart_pirate_logo[] = {
+            0x00,0x00,0x00,0x00,
+            0x00,0x00,0x00,0x00,
+            0x00,0x1F,0xF8,0x00,
+            0x00,0x01,0x80,0x00,
+            0x00,0x01,0x80,0x00,
+            0x02,0x0F,0xF0,0x40,
+            0x04,0x38,0x1C,0x20,
+            0x08,0x60,0x06,0x10,
+            0x08,0x80,0x01,0x10,
+            0x19,0x00,0x00,0xF8,
+            0x36,0x78,0x1E,0x68,
+            0x26,0xFC,0x3F,0x64,
+            0x24,0xFC,0x3F,0x24,
+            0x64,0xFC,0x3F,0x24,
+            0x04,0x7C,0x3E,0x20,
+            0x04,0x38,0x1C,0x20,
+            0x04,0x00,0x80,0x20,
+            0x04,0x00,0x00,0x20,
+            0x07,0xFF,0xFF,0xE0,
+            0x24,0xC9,0x93,0x24,
+            0x26,0x08,0x10,0x24,
+            0x26,0x08,0x10,0x6C,
+            0x1F,0x88,0x11,0xF8,
+            0x19,0x88,0x11,0x98,
+            0x08,0xC8,0x13,0x10,
+            0x04,0x78,0x1E,0x20,
+            0x06,0x1D,0xB8,0x60,
+            0x00,0x07,0xE0,0x00,
+            0x00,0x01,0x80,0x00,
+            0x00,0x19,0x98,0x00,
+            0x00,0x07,0xE0,0x00,
+            0x00,0x00,0x00,0x00
+        };
+    oled_write_raw_P(heart_pirate_logo, sizeof(heart_pirate_logo));*/
+
+
+    static const char PROGMEM hell_logo[] = {
+            0x00, 0x80, 0xc0, 0xc0,
+            0x60, 0x60, 0x30, 0x30,
+            0x18, 0x1c, 0x0c, 0x00,
+            0x00, 0x00, 0x00, 0x80,
+            0xe0, 0x78, 0x1e, 0x06,
+            0x00, 0x0c, 0x1c, 0x18,
+            0x30, 0x30, 0x60, 0x60,
+            0xc0, 0xc0, 0x80, 0x00,
+            0x01, 0x03, 0x07, 0x06,
+            0x0c, 0x0c, 0x18, 0x18,
+            0x30, 0x70, 0x60, 0x00,
+            0xc0, 0xf0, 0x3c, 0x0f,
+            0x03, 0x00, 0x00, 0x00,
+            0x00, 0x60, 0x70, 0x30,
+            0x18, 0x18, 0x0c, 0x0c,
+            0x06, 0x07, 0x03, 0x01,
+            0x00, 0xf8, 0xf8, 0x80,
+            0x80, 0x80, 0xf8, 0xf8,
+            0x00, 0x80, 0xc0, 0xc0,
+            0x40, 0xc0, 0xc0, 0x80,
+            0x00, 0xf8, 0xf8, 0x00,
+            0xf8, 0xf8, 0x00, 0x08,
+            0x38, 0x08, 0x00, 0x38,
+            0x08, 0x30, 0x08, 0x38,
+            0x00, 0x1f, 0x1f, 0x01,
+            0x01, 0x01, 0x1f, 0x1f,
+            0x00, 0x0f, 0x1f, 0x1a,
+            0x12, 0x1a, 0x1b, 0x0b,
+            0x00, 0x1f, 0x1f, 0x00,
+            0x1f, 0x1f, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00
+        };
+    oled_write_raw_P(hell_logo, sizeof(hell_logo));
+}
+
+static void print_logo_narrow(void) {
+    render_logo();
+
+    /* wpm counter */
+    // char wpm_str[5];
+    // wpm_str[4] = '\0';
+    // wpm_str[3] = '0' + current_wpm % 10;
+    // wpm_str[2] = '0' + ( current_wpm /= 10) % 10;
+    // wpm_str[1] = '0' + current_wpm / 10;
+    // wpm_str[0] = ' ';
+    oled_set_cursor(0,13);
+    char wpm_str[10]; // if not working, maybe moving it to line ~20ish
+    sprintf(wpm_str, "       WPM: %03d", get_current_wpm());
```

Comment:
https://docs.qmk.fm/#/squeezing_avr?id=oled-tweaks

Eg, this will use ~1kb less.
```suggestion
    oled_write_P(PSTR("      WPM: "), false);
    oled_write(get_u8_str(get_current_wpm(), '0'), false);
```